### ISSUE TITLE
[MAS2.4.7] [Keyboard Navigation-Help] Visual focus indicator is not clearly visible for “Jump to bot state inspection and Inspection middleware” links 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1808]
**Actual Result:**
Visual focus indicator is not clearly visible for “Jump to bot state inspection and Inspection middleware” links

**Expected Result:**
Visual focus indicator should be clearly visible for “Jump to bot state inspection and Inspection middleware” links

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the help menu and select it 
3.Navigate to the get started with  channels(bot inspector) and select it  
4.get started with the bot inspector screen gets appears 
5.Navigate to the  “Jump to bot state inspection and Inspection middleware” links
6.Observe the issue 

**User Impact:**
Users who rely on Keyboard navigation will be affected the keyboard focus indicator is not clearly visible on control so that keyboard dependant users will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}
[MAS2.4.7_Help.zip](https://github.com/microsoft/BotFramework-Emulator/files/3562227/MAS2.4.7_Help.zip)
